### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by Sayanta66**
Fixed internal broken links ...Reference PR #4915

### CODE REVIEW

**Comment by Sayanta66** [comment]
@justinegeffen give the docathon label

File: site/content/integrate/admin-guide/admin-slash-commands/_index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: "Mattermost supports slash commands to easily integrate external ap
 weight: 80
 ---
 
-Messages that begin with `/` are interpreted as slash commands. The commands will send an HTTP POST request to a web service, and process a response back to Mattermost. Mattermost supports both [built-in](https://docs.mattermost.com/developer/slash-commands.html#built-in-commands) and [custom slash commands](https://docs.mattermost.com/developer/slash-commands.html#custom-slash-command).
+Messages that begin with `/` are interpreted as slash commands. The commands will send an HTTP POST request to a web service, and process a response back to Mattermost. Mattermost supports both [built-in](https://docs.mattermost.com/messaging/executing-slash-commands.html#built-in-commands) and [custom slash commands](https://docs.mattermost.com/developer/slash-commands.html#custom-slash-command).
```
Comment (justinegeffen) [inline site/content/integrate/admin-guide/admin-slash-commands/_index.md:8]:
```suggestion
Messages that begin with `/` are interpreted as slash commands. The commands will send an HTTP POST request to a web service, and process a response back to Mattermost. Mattermost supports both [built-in](#built-in-commands) and [custom slash commands](#custom-slash-command).
```

File: site/content/integrate/admin-guide/admin-slash-commands/_index.md [hunk 2]
```diff
@@ -56,11 +56,11 @@ You can follow these general guidelines to set up a custom Mattermost slash comm
 
 1. Open **Main Menu > Integrations > Slash Commands**. If you don't have the **Integrations** option in your Main Menu, slash commands may not be enabled on your Mattermost server or may be disabled for non-admins. Enable them from **System Console > Integrations > Integration Management** or ask your Mattermost System Admin to do so.
 2. Select **Add Slash Command** and add a name and description for the command.
-3. Set the **Command Trigger Word**. The trigger word must be unique and cannot begin with a slash or contain any spaces. It also cannot be one of the [built-in commands](https://docs.mattermost.com/help/messaging/executing-commands.html#built-in-commands).
+3. Set the **Command Trigger Word**. The trigger word must be unique and cannot begin with a slash or contain any spaces. It also cannot be one of the [built-in commands](https://docs.mattermost.com/messaging/executing-slash-commands.html#built-in-commands).
```
Comment (justinegeffen) [inline site/content/integrate/admin-guide/admin-slash-commands/_index.md:59]:
```suggestion
3. Set the **Command Trigger Word**. The trigger word must be unique and cannot begin with a slash or contain any spaces. It also cannot be one of the [built-in commands](#built-in-commands).
```

File: site/content/integrate/admin-guide/admin-slash-commands/_index.md [hunk 3]
```diff
@@ -124,23 +124,23 @@ which would render in Mattermost as:
...
-4. Mattermost outgoing webhooks are Slack-compatible. You can copy-and-paste code used for a Slack outgoing webhook to create Mattermost integrations. Mattermost [automatically translates the Slack's proprietary JSON payload format](https://docs.mattermost.com/developer/webhooks-incoming.html#translate-slack-s-data-format-to-mattermost).
+2. Posts size is limited to 16393 characters for servers running [Mattermost Server v5.0 or later](https://docs.mattermost.com/upgrade/important-upgrade-notes.html). Use the [extra_responses](#parameters) field to reply to a triggered slash command with more than one post.
+3. You can restrict who can create slash commands in [System Console > Integrations > Integration Management](https://docs.mattermost.com/configure/configuration-settings.html#restrict-managing-integrations-to-admins).
+4. Mattermost outgoing webhooks are Slack-compatible. You can copy-and-paste code used for a Slack outgoing webhook to create Mattermost integrations. Mattermost [automatically translates the Slack's proprietary JSON payload format](https://docs.mattermost.com/developer/slash-commands.html#translate-slack-s-data-format-to-mattermost).
```
Comment (justinegeffen) [inline site/content/integrate/admin-guide/admin-slash-commands/_index.md:136]:
```suggestion
4. Mattermost outgoing webhooks are Slack-compatible. You can copy-and-paste code used for a Slack outgoing webhook to create Mattermost integrations. Mattermost [automatically translates the Slack's proprietary JSON payload format](#translate-slack-s-data-format-to-mattermost).
```

**Review by justinegeffen — COMMENTED** [review]
Thanks for this PR. There are some instances where the URL is invalid as the reference is to this doc itself. I've changed those and you can commit the changes once you've reviewed them. :)

**Comment by ArmanChand** [comment]
Thanks for your efforts for this PR. But it seems that @Rutam21 has already raised a PR
requesting the exact same changes .Here is the link : 
https://github.com/mattermost/mattermost-
developer-documentation/pull/874

**Comment by justinegeffen** [comment]
Thanks for noting that, @ArmanChand! Much appreciated. :)

**Pull request closed without merging**